src/MockPlayerBridge.ts

```diff
@@ -41,23 +73,157 @@ export class MockPlayerBridge extends AbstractPlayerBridge {
     return 'dummy-stream-id';
   }
 
-  protected subscribeFullscreen(listener: (value: boolean) => void) {
-    // Note: LiveryPlayer only listens to itself becoming fullscreen; not just anything
-    document.addEventListener('fullscreenchange', () => {
-      listener(!!document.fullscreenElement);
+  protected pause() {
+    this.setPlaybackState('PAUSED');
+  }
+
+  protected play() {
+    this.setPlaybackState('PLAYING');
+  }
+
+  protected reload() {
+    this.setPlaybackState('PLAYING');
+  }
+
+  protected seek(position: number) {
+    this.setPlaybackState('SEEKING');
+    return position;
+  }
+
+  protected selectQuality(index: number) {
+    return index;
+  }
+
+  protected setControlsDisabled(disabled: boolean) {
+    return disabled;
+  }
+
+  protected setDisplay(display: DisplayMode) {
+    return this.setDisplayState(display);
+  }
+
+  protected setMuted(muted: boolean) {
+    return this.setMutedState(muted);
+  }
+
+  protected submitUserFeedback(payload: UserFeedbackPayload) {
+    return payload;
+  }
+
+  protected subscribeConfig(listener: (value: Config) => void) {
+    const config: Config = {
+      controls: {
+        cast: true,
+        contact: true,
+        error: true,
+        fullscreen: true,
+        mute: true,
+        pip: true,
+        play: true,
+        quality: true,
+        scrubber: true,
+      },
+      customerId: 'id',
+      streamPhase: 'PRE',
+      streamPhases: ['LIVE', 'POST', 'PRE'],
```

Comment:
This is not a valid streamPhases list, see ticket, it should be: `Record<number, StreamPhase>`. Where the numbers are the unix timestamps (e.g: `Date.now()`) when the phase started.

Reply:
Huh that's funny, I thought Typescript would've errored here but I guess StreamPhase[] does match Record<number, StreamPhase> 🤔

Reply:
Yeah, it's probably a poor choice of data structure, at least as far as JavaScript and TypeScript are concerned. But this is what the native player and server (it's passed via remote config) devs came up with so we're kind of stuck with it. I also don't feel like restructuring this internally in our web player as it might get confusing then.